.travis.yml

```diff
@@ -45,21 +45,12 @@ jobs:
           skip_cleanup: true
           on:
             branch: release
-        - provider: pages
-          skip_cleanup: true
-          committer-from-gh: true
-          keep-history: false
-          local-dir: dist/gh_deploy
-          github-token: $GITHUB_TOKEN
-          on:
-            branch: release
         - provider: s3
           access_key_id: $AWS_ACCESS_KEY_ID
           secret_access_key: $AWS_SECRET_ACCESS_KEY
           bucket: "harp.gl"
           skip_cleanup: true
           local_dir: dist/s3_deploy
-          upload-dir: docs
```

Comment:
Does this mean the documentation will be in the root folder of the s3 bucket? Do we have to change all links to the documentation then?
I think it would be better to keep the docs folder b/c we might put other stuff into that bucket as well

Reply:
Documentation remains in docs/ but since we deploy all together to s3 we need to put website files to root dir. This change is synced with change in prepare_doc_deploy.ts where we now put documentation into s3_deploy/docs instead of just s3_deploy.

Reply:
okay, got it.